config/translations/console.fr.yml

```diff
@@ -180,18 +180,18 @@ commands:
                 routing: 'Mise à jour des routes'
         module:
             description: 'Générer un module.'
-            help: 'Le <info> générer: le module </ info> commande aide à générer un nouveau module'
+            help: 'La commande <info>generate:module</info> vous assiste dans la génération d''un nouveau module'
             welcome: 'Bienvenue sur le générateur de modules Drupal'
             options:
                 module: 'Le nom du module'
-                machine-name: 'Le nom de la machine (en minuscules et soulignent seulement)'
+                machine-name: 'Le nom machine (minuscules et underscore _ seulement)'
                 module-path: 'Le chemin du module'
                 description: 'Description du module'
-                core: 'Version de core.'
-                package: 'Paquet Module'
+                core: 'Version de core'
+                package: 'Paquet (<info>package</info>) du module'
                 controller: 'Contrôleur par défaut'
-                composer: 'Add a composer.json file'
-                dependencies: 'Module dependencies (i.e. context, galleria, panels)'
+                composer: 'Ajouter un fichier composer.json'
+                dependencies: 'Dépendences du modules (par exemple : context, galleria, panels)'
```

Comment:
"Dépendences" should be "Dépendance" with a "a"
"modules" should not take a "s"